web/src/test/java/com/epam/rft/atsy/helper/MultipartFileCreatorTestHelper.java

```diff
@@ -10,9 +10,9 @@
   public static final String MULTIPART_FILE_NAME = "file";
   public static final String CONTENT_TYPE = "text/plain";
 
-  public static MockMultipartFile createMultipartFile(String originalFilename, int fileSize) {
+  public static MockMultipartFile createMultipartFile(String originalFilename, Long fileSizeByte) {
     return new MockMultipartFile(MULTIPART_FILE_NAME, originalFilename, CONTENT_TYPE,
-        getStringBySize(fileSize).getBytes());
+        getStringBySize(fileSizeByte.intValue()).getBytes());
```

Comment:
It is a very dangerous solution because long can store a wider range than an integer, so intValue method will truncate those numbers, which are bigger then the biggest integer number, without any exception, so this createMultipartFile method simply will return a wrong result in case of "enough" big long fileSizeByte parameter value. Let's give it a try, call this method with 105.000.000.000 fileSizeByte parameter value, the result will not be the expected. Anyway this is a bad practice: you use a long parameter as an integer, but why? If you want to use this parameter as an integer, then the parameter type have to be integer or if it is a requirement to support long, then do it in a right and safety way.